app/src/pages/inside/manualLaunchesPage/manualLaunchesFilterSidePanel/manualLaunchesFilterSidePanel.scss

```
@@ -0,0 +1,150 @@
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.filter-side-panel {
+  width: 460px;
+  z-index: $Z-INDEX-OVERLAY-PANEL;
+
+  :global([class*='header-section']) {
```

Stylelint (17.5.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)